test/github-issues/85/issue-85.ts

```diff
@@ -0,0 +1,35 @@
+import {Connection} from "../../../src";
+import {closeTestingConnections, createTestingConnections, reloadTestingDatabases} from "../../utils/test-utils";
+import {Document} from "./entity/Document";
+import {expect} from "chai";
+
+describe("github issues > #85 - Column option insert: false, update: false", () => {
+
+    let connections: Connection[];
+
+    before(async () => connections = await createTestingConnections({
+        entities: [__dirname + "/entity/*{.js,.ts}"],
+        schemaCreate: true,
+        dropSchema: true
+    }));
+    beforeEach(() => reloadTestingDatabases(connections));
+    after(() => closeTestingConnections(connections));
+
+  it("should be able to have a non-inserted Column", () => Promise.all(connections.map(async connection => {
+    const queryRunner = connection.createQueryRunner();
```

Comment:
you **must** release queryRunner or CI will fail sometimes

Reply:
also, not sure Ive got meaning of this test. If you don't want to allow values to be inserted I guess you need to set a `DEFAULT` or don't set anything at all and write some value into entity before calling `save` then assert if value isn't equal to the value you wrote and equal to DEFAULT or null if you didn't set a default.

Reply:
Thanks for the tip, I've released the `queryRunner`.  I also added a simpler, more intuitive test, and then documented my somewhat weirder one a bit better (I hope).